### ISSUE TITLE
Some checks failing with newest version of Rubocop

### ISSUE THREAD

**Issue by jordynchuhaloff** [brigade/haml-lint#113]
After upgrading to Rubocop (0.37.0), we're seeing these errors:
```
choose_upload_method.html.haml:1 [W] RuboCop: unexpected token kDO
(Using Ruby 2.1 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)
choose_upload_method.html.haml:2 [W] RuboCop: unexpected token tSYMBOL
(Using Ruby 2.1 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)
```

Using this code to repro:
```haml
= form_for @wizard do |form|
  = form.hidden_field :name
```

**Comment by sds** [comment]
Thanks for the report, @jordynchuhaloff.

The example you provided gets converted to the following Ruby when passed to RuboCop:

```ruby
form_for @wizard do |form|
  form.hidden_field :name
end
```

When I run RuboCop 0.37.0 against the above Ruby, I get the same errors you report. Upgrading to RuboCop 0.37.1 fixes the issue, so this was a problem with RuboCop. Hope that helps!

**Comment by jordynchuhaloff** [comment]
Updated rubocop and it works for me too. Thank you!!
